src/main/java/com/targomo/client/api/exception/TargomoClientException.java

```diff
@@ -1,14 +1,34 @@
 package com.targomo.client.api.exception;
 
+import com.targomo.client.Constants;
+import lombok.Getter;
+
 public class TargomoClientException extends Exception {
 
 	private static final long serialVersionUID = 748077356312331362L;
+	
+	@Getter
+	private int httpStatusCode;
 
 	public TargomoClientException(String message, Throwable throwable) {
-		super(message, throwable);
+		this(message, throwable, Constants.BAD_GATEWAY_CODE);
```

Comment:
would be nice to have a small comment here: Default is that this Exception is tarnslated into a BadGateway response (or something like that)

Reply:
Updated and added full javadoc